docs/release_1.2/release_notes_1.2.md

```diff
@@ -0,0 +1,33 @@
+# Avengers Assemble v1.2 Release Notes
+
+With our additions, we are confident that Avengers Assemble has a MVP. Although some features require refinement, we have implemented enough features with sufficient quality to ensure a working product for head TAs for CS1101S.
+## What's New
+### 1. Export Feature
+Avengers Assemble now supports exporting data of up to 500 contacts to a CSV file!
+
+1. Simply type `export` to export a CSV file of the contacts. 
+<img src="export_1.png" alt="Typing in the `export` command.">
+
+2. Once this message shows, the CSV is available for external use by the head TA.
+<img src="export_2.png" alt="The CSV will be exported.">
+
+3. An example of the exported CSV file.
+<img src="export_csv_result.png" alt="The CSV file.">
+
+### 2. Find Feature
```

Comment:
Can talk about the copy feature here too! You just need screenshots for this, but the text can be something like the text below!

### 3. Copy Feature
Avengers Assemble also supports copying the emails of any specific group of contacts in the list into your clipboard!

1. Use the `find` command to search for the people whose emails you would like to copy.
2. Type `copy` to copy all the emails of currently displayed contacts into your clipboard.
3. All emails will be copied into your clipboard!